.travis.yml

```diff
@@ -24,9 +25,11 @@ before_install:
   - ln -s $HOME/islandora_solr_search sites/all/modules/islandora_solr_search
   - ln -s $TRAVIS_BUILD_DIR sites/all/modules/islandora_solr_metadata
   - drush en --user=1 --yes islandora_solr_metadata
+before_script:
```

Comment:
Would prefer if you removed this full chunk. This is not part of this ticket and seems more like an issue with the rebase.